New package: EnergySamplers v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: EnergySamplers
- Repository: https://github.com/JuliaTrustworthyAI/EnergySamplers.jl
- Created by: @pat-alt
- Version: v1.0.0
- Commit: 20bda5cb21339e0a60f99a6dbaeab878fb5ad35a
- Reviewed by: @pat-alt
- Reference: https://github.com/JuliaTrustworthyAI/EnergySamplers.jl/commit/20bda5cb21339e0a60f99a6dbaeab878fb5ad35a#commitcomment-146338842
- Description: A small Julia package for energy-based sampling.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd69e00b621d9b06d45c5182b2093ce7db2ac50bf6e1250bf56072bd7b0ee4cd9f9efcf7cb4fdecf6ce1b8c660c993d4a4a56cdba7773be873981201ceb8b5b542efd7c756dc3a6a7bf1cf8016fad97fa9df68aa47d1022364372f8c96aa2c03bc19d736d9e7795edf2ab9624cf19f11d473b346a18ae91c77a2aba8d57847707e8894ac159d7bc4c011c4c81b5b551c6c534288d0770d2b4490f9aafbf38ffacf7cbbb752106cb925a0d1de8cefaea3ec5acb63f73e58289c410a119f0f160b975 -->

goerz commented:
Could you add a paragraph or two to the README and/or documentation that gives some context for this package? What kind of problem does it solve, what approach it implements, stuff like that?

pat-alt commented:
@goerz thanks for the comment and, yes, absolutely, that was sloppy of me. Working on it now and will update the PR asap.

JuliaRegistrator committed:
New package: EnergySamplers v1.0.0

UUID: f446124b-5d5e-4171-a6dd-a1d99768d3ce
Repo: https://github.com/JuliaTrustworthyAI/EnergySamplers.jl.git
Tree: 6072d358f2208c7cf56c18d4112d94f8660fa4b0

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
[noblock] Thanks! You'll want to make sure to edit https://github.com/JuliaRegistries/General/pull/114601#issuecomment-2333499421 so that it includes `[noblock]`

pat-alt commented:
[noblock] all done, thanks again!
